core/src/Borrower.sol

```diff
@@ -140,21 +140,21 @@ contract Borrower is IUniswapV3MintCallback {
                 // If both are zero or neither is zero, there's nothing more to do.
                 // Callbacks/swaps won't help.
             } else if (liabilities0 > 0) {
-                uint256 converted0 = liabilities0 / strain;
-                uint256 maxLoss1 = Math.mulDiv(converted0, priceX96, Q96) + incentive1 / strain;
+                uint256 expected0 = liabilities0 / strain;
+                uint256 advance1 = Math.mulDiv(expected0, priceX96, Q96) + incentive1 / strain;
```

Comment:
Valid choice. I've said it in other comments, but as a reminder: subject to potential change per https://github.com/aloelabs/aloe-ii/pull/30#discussion_r1073222041.